core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -1,11 +1,11 @@
 /**
- * Copyright (c) 2010 Yahoo! Inc. All rights reserved. 
- * 
+ * Copyright (c) 2010 Yahoo! Inc. All rights reserved.
```

Comment:
note to other reviewers, this copyright header is wrong but the problem is covered by #313.